artiq/frontend/artiq_ctlmgr.py

```diff
@@ -60,25 +80,27 @@ def main():
         loop = asyncio.get_event_loop()
     atexit.register(loop.close)
 
-    logfwd = LogForwarder(args.server, args.port_logging,
-                          args.retry_master)
+    # Set up log forwarding to remote PC/master
+    logfwd = LogForwarder(args.server, args.port_logging, args.retry_master)
     logfwd.addFilter(source_adder)
     root_logger.addHandler(logfwd)
     logfwd.start()
     atexit_register_coroutine(logfwd.stop)
 
-    ctlmgr = ControllerManager(args.server, args.port_notify,
-                               args.retry_master)
+    # Start a Controller Manager. Waits to start devices until requested by Master.
+    ctlmgr = ControllerManager(args.server, args.port_notify, args.retry_master)
     ctlmgr.start()
     atexit_register_coroutine(ctlmgr.stop)
 
     class CtlMgrRPC:
         retry_now = ctlmgr.retry_now
 
+    # Allow remote retry_now procedure calls to the controller manager.
     rpc_target = CtlMgrRPC()
     rpc_server = Server({"ctlmgr": rpc_target}, builtin_terminate=True)
-    loop.run_until_complete(rpc_server.start(bind_address_from_args(args),
-                                             args.port_control))
+    loop.run_until_complete(
+        rpc_server.start(bind_address_from_args(args), args.port_control)
+    )
```

Comment:
Can you keep PRs to a single topic?

Reply:
Ok. So what is your recommendation for scope of a single PR? I try to fix any unreadable code that I discover while working on a single task, and it gets wrapped into the PR. This keeps me from making lots of PRs on just documentation/refactoring, and it reduces my workload in writing infinite amounts of PRs. Because fixing code smells and incomprehensible code in ARTIQ is not my primary job.

Reply:
Removed all new documentation/comments in https://github.com/m-labs/artiq/pull/1181/commits/b9d50a9a955bebf52472860186f549fb438987d6.

Reply:
This isn't incomprehensible and not even code smell as this style is in PEP8.
https://www.python.org/dev/peps/pep-0008/#indentation first example.